recipes/heyoka/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: heyoka
+  version: {{ version }}
+
+source:
+  url: https://github.com/bluescarni/heyoka/archive/v{{ version }}.tar.gz
+  sha256: 4f82918fac1c269fe32c1078fcace31614fb329070d729ef7d61e5655129ebbe
+
+build:
+  number: 0
+  run_exports:
+    # SO name changes at minor rev bumps.
+    - {{ pin_subpackage('heyoka', max_pin='x.x') }}
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

should you skip the ``compiler`` on windows as you are using ``clang`` ?
```suggestion
    - {{ compiler('c') }}  # [unix]
    - {{ compiler('cxx') }}  # [unix]
```

---

Ok, thanks for the suggestion!